Makefile.js

```diff
@@ -486,6 +511,7 @@ target.changelog = function() {
     rm("CHANGELOG.tmp");
     rm("CHANGELOG.md");
     mv("CHANGELOG.md.tmp", "CHANGELOG.md");
+    return output;
```

Comment:
@gyandeeps See my comment above. I suggested making this change to get fully formatted changelog message.